app/views/feeds/_form_distribution.html.erb

```diff
@@ -31,13 +31,23 @@
         </div>
       </div>
 
-      <div class="col-12 mb-4">
-        <div class="form-floating input-group">
-          <%= form.text_field :published_url, value: feed.published_url(false), disabled: true %>
-          <%= form.label :published_url %>
-          <%= field_link feed.published_url(true) %>
+      <% if feed.persisted? %>
```

Comment:
It doesn't make sense to show these until you create the Feed.  Since the path requires a `feed.slug`.